Formula/isl.rb

```diff
@@ -32,31 +31,46 @@ class Isl < Formula
     depends_on "libtool" => :build
   end
 
+  depends_on "llvm" => :build
+  depends_on "python@3.9" => [:build, :test]
+
   depends_on "gmp"
 
   def install
+    python = Formula["python@3.9"]
+    site_packages = Language::Python.site_packages(python)
```

Comment:
```suggestion
    site_packages = Language::Python.site_packages("python3")
```
This works too, and makes the formula easier to update for `python@3.10`.